addon/components/hyper-table-v2/index.ts

```diff
@@ -122,4 +122,9 @@ export default class HyperTableV2 extends Component<HyperTableV2Args> {
   onRowHover(row: Row, hovered: boolean) {
     set(this.args.handler.rows[this.args.handler.rows.indexOf(row)], 'hovered', hovered);
   }
+
+  @action
+  reloadPage() {
+    window.location.reload();
```

Comment:
Well this error is only supposed to be shown when there's an issue with the backend ; I'd rather have the whole app reload at that point instead of re-triggering the flow separately (there is now way to retrigger the whole thing easily as far as I know).
Plus, it's not going to happen often right ? 😄